extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.ts

```diff
@@ -260,43 +254,20 @@ export class DragAndDropSortInputValidationService {
 
         for (var k = 0; k < ranges.length; k++) {
           var earlierRule = answerGroups[ranges[k].answerGroupIndex].
-            getRulesAsList()[ranges[k].ruleIndex];
```

Comment:
Why these changes below? Seems unrelated?

Reply:
Doesn't this properly revert the changes from #10246? https://github.com/oppia/oppia/pull/10246/files#diff-3d7e1efacf316f35426e24bedbd89564L255